### DISCUSSION TITLE
HTTP Default timeout error

### DISCUSSION THREAD

**Discussion by Pavithrachan** [artilleryio/artillery#2634] [Artillery]
Hi , 
I am trying to perform API- load test/stress tests with Artillery . 15 min - 1 hr load tests working fine with Arrival Rate 2. But A.R above 2 causing timeout Error . i am not able to override Artillery's 10000ms default timeout. below are the load profiles i tried.

    - duration: 3600
      arrivalRate: 3
      name: Load test
    - duration: 3600
       arrivalRate: 5
      name-Stress Test
    - duration: 300
       arrivalRate: 10
       name : spike test  
    - duration: 3600
       arrivalRate: 1 
       rampTo: 4
       name: Scaling load

I have added timeouts as 180, 300 as some of the requests might take around 30 sec to respond based on our target system behaviour. But still i could not override the 10000ms timeout.
<img width="485" alt="ArtilleryTimeout" src="https://github.com/artilleryio/artillery/assets/166719619/0eea3a30-acc8-4847-ad0d-44c48ff8d0aa">

Please suggest if any other ways to make artillery wait more than 10000ms to receive response from the server. Thanks in Advance.

**Comment by bernardobridge** [comment]
Hi @Pavithrachan .

You are using the wrong name for config option: https://www.artillery.io/docs/reference/engines/http#request-timeouts

The config key is called `http` not `https`. 

Let me know if that works.

**Reply by Pavithrachan** [reply]
Great, I changed the config key as you mentioned it's working now !!
Previously i have used' https'  because my target host url is 'https'. So, i thought the timeout config is based on our target host . 

 Thanks @bernardobridge  :)
